src/msw/statbox.cpp

```diff
@@ -39,6 +39,9 @@
 #include "wx/sysopt.h"
 
 #include "wx/msw/uxtheme.h"
+
+#include <windowsx.h> // needed by GET_X_LPARAM and GET_Y_LPARAM macros
```

Comment:
Including it from `wx/msw/private.h` results in at least one apparently unrelated error:
> ..\..\src\msw\gdiplus.cpp(916): error C2039: 'DeleteObject' : is not a member of 'wxGdiPlus'
>           ..\..\src\msw\gdiplus.cpp(759) : see declaration of 'wxGdiPlus'
> ..\..\src\msw\gdiplus.cpp(916): error C2440: 'return' : cannot convert from 'int' to 'Gdiplus::GpStatus'
>           Conversion to enumeration type requires an explicit cast (static_cast, C-style cast or function-style cast)

Including `wx/msw/winundef.h` too did not change anything.
Should it be reverted ?